test/services/pilot/deploy_mocks.sh

```diff
@@ -47,7 +47,6 @@ mocks_create() {
 
 undeploy() {
         mocks_delete
-        kube_cluster_info
```

Comment:
why do you remove the cluster info after deployment? Mocks will be deployed separately from the manager. Is there another place where you ask for cluster info? if not why you prefer to remove this part of deployment?

Reply:
added kube_cluster_info back to the script. Thanks.